### PR TITLE
Fix #177367 sakisiru.jp

### LINKED ISSUE #177367: sakisiru.jp
### Issue URL (Annoyance)
 [https://sakisiru.jp/48207](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsakisiru.jp%2F48207)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for iOS v4.5.2
DNS filtering: | server: `https://doh.opendns.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by artemiv4nov**
Fix for #177367

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -8732,3 +8732,4 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_apps
 !+ NOT_PLATFORM(windows, mac, android, ios, ext_ublock) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 !
+sakisiru.jp##.newsletter
```
Comment (piquark6046) [inline JapaneseFilter/sections/specific.txt:8735]: please add the domain into `tastesoflizzyt.com##.newsletter` at `AnnoyancesFilter/Popups/sections/popups_specific.txt` and remove the current rule.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Please rename title of the PR to `Fix #177367 sakisiru.jp`.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (artemiv4nov) [reply JapaneseFilter/sections/specific.txt]: Done

**Review by piquark6046 — CHANGES_REQUESTED** [review]
I meant that adding the `sakisiru.jp` domain into https://github.com/AdguardTeam/AdguardFilters/blob/4afe0fd8be364a0b2764d8d20cd2f85e1be2e648/AnnoyancesFilter/Popups/sections/popups_specific.txt#L4215.
Not adding new rule.

**Review by piquark6046 — APPROVED** [review]

**Review by Alex-302 — CHANGES_REQUESTED** [review]
Wrong file
https://github.com/AdguardTeam/AdguardFilters/blob/cd6d692fe236fe3fa8a86460805d9747e7ca6306/AnnoyancesFilter/Popups/sections/subscriptions_specific.txt#L13

**Comment by Alex-302** [comment]
@piquark6046 Better don't fix errors in PRs of filters day.

**Comment by piquark6046** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/177783#issuecomment-2074852282 Okay.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**